### ISSUE TITLE
CUDA Error: an illegal memory access was encountered: No error

### ISSUE THREAD

**Issue by Pratyush-DS** [AlexeyAB/darknet#8027]
conv    256       3 x 3/ 1     26 x  26 x 256 ->   26 x  26 x 256 0.797 BF
conv    256       3 x 3/ 1     26 x  26 x 256 ->   26 x  26 x 256 0.797 BF
lstm_activation: Using default 'tanh'
 150 conv     24       1 x 1/ 1     26 x  26 x 256 ->   26 x  26 x  24 0.008 BF
 151 yolo
[yolo] params: iou loss: ciou (4), iou_norm: 0.07, obj_norm: 1.00, cls_norm: 1.00, delta_norm: 1.00, scale_x_y: 1.10
nms_kind: greedynms (1), beta = 0.600000
 152 route  148                                            ->   26 x  26 x 512
 153 conv    512       3 x 3/ 2     26 x  26 x 512 ->   13 x  13 x 512 0.797 BF
 154 route  153 117                                ->   13 x  13 x 768
 155 conv    512       1 x 1/ 1     13 x  13 x 768 ->   13 x  13 x 512 0.133 BF
 156 conv   1024       3 x 3/ 1     13 x  13 x 512 ->   13 x  13 x1024 1.595 BF
 157 conv    512       1 x 1/ 1     13 x  13 x1024 ->   13 x  13 x 512 0.177 BF
 158 conv   1024       3 x 3/ 1     13 x  13 x 512 ->   13 x  13 x1024 1.595 BF
 159 conv    512       1 x 1/ 1     13 x  13 x1024 ->   13 x  13 x 512 0.177 BF
 160 conv   1024       3 x 3/ 1     13 x  13 x 512 ->   13 x  13 x1024 1.595 BF
 161 CONV_LSTM Layer: 13 x 13 x 1024 image, 256 filters
conv    256       3 x 3/ 1     13 x  13 x1024 ->   13 x  13 x 256 0.797 BF
conv    256       3 x 3/ 1     13 x  13 x1024 ->   13 x  13 x 256 0.797 BF
conv    256       3 x 3/ 1     13 x  13 x1024 ->   13 x  13 x 256 0.797 BF
conv    256       3 x 3/ 1     13 x  13 x1024 ->   13 x  13 x 256 0.797 BF
conv    256       3 x 3/ 1     13 x  13 x 256 ->   13 x  13 x 256 0.199 BF
conv    256       3 x 3/ 1     13 x  13 x 256 ->   13 x  13 x 256 0.199 BF
conv    256       3 x 3/ 1     13 x  13 x 256 ->   13 x  13 x 256 0.199 BF
conv    256       3 x 3/ 1     13 x  13 x 256 ->   13 x  13 x 256 0.199 BF
lstm_activation: Using default 'tanh'
 162 conv     24       1 x 1/ 1     13 x  13 x 256 ->   13 x  13 x  24 0.002 BF
 163 yolo
[yolo] params: iou loss: ciou (4), iou_norm: 0.07, obj_norm: 1.00, cls_norm: 1.00, delta_norm: 1.00, scale_x_y: 1.05
nms_kind: greedynms (1), beta = 0.600000
Total BFLOPS 73.476
avg_outputs = 486175
 Allocate additional workspace_size = 33.46 MB
Loading weights from yolov4.conv.137...
 seen 64, trained: 0 K-images (0 Kilo-batches_64)
Done! Loaded 137 layers from weights-file
Learning Rate: 0.001, Momentum: 0.949, Decay: 0.0005
 Detection layer: 139 - type = 28
 Detection layer: 151 - type = 28
 Detection layer: 163 - type = 28
 If error occurs - run training with flag: -dont_show
Resizing, random_coef = 1.40

 608 x 608
 Create 6 permanent cpu-threads
 try to allocate additional workspace_size = 71.26 MB
 CUDA allocate done!
Loaded: 0.000000 seconds
v3 (iou loss, Normalizer: (iou: 0.07, obj: 1.00, cls: 1.00) Region 139 Avg (IOU: 0.327542), count: 12, class_loss = 3496.552734, iou_loss = 32.626465, total_loss = 3529.179199
 Create stream 0
 Create cudnn-handle 0
v3 (iou loss, Normalizer: (iou: 0.07, obj: 1.00, cls: 1.00) Region 151 Avg (IOU: 0.356842), count: 7, class_loss = 1071.564819, iou_loss = 6.713623, total_loss = 1078.278442
v3 (iou loss, Normalizer: (iou: 0.07, obj: 1.00, cls: 1.00) Region 163 Avg (IOU: 0.000000), count: 1, class_loss = 288.012543, iou_loss = 0.000000, total_loss = 288.012543
 total_bbox = 19, rewritten_bbox = 0.000000 %
CUDA status Error: file: C:\dark\darknet-master\darknet-master\src\dark_cuda.c : cuda_push_array() : line: 589 : build time: Aug 12 2021 - 20:06:17

 CUDA Error: an illegal memory access was encountered

Darknet error location: C:\dark\darknet-master\darknet-master\src\dark_cuda.c, check_error, line #70
CUDA Error: an illegal memory access was encountered: No error
